crates/mun_target/src/spec/apple_sdk_base.rs

```diff
@@ -0,0 +1,49 @@
+use crate::spec::TargetOptions;
+
+use Arch::*;
+
+#[allow(non_camel_case_types, dead_code)]
```

Comment:
Is this `dead_code`? It seems to be used underneath

Reply:
The enum itself is used, but not all variants are constructed. I kept them in because these are all the supported architectures at the moment.